scripts/start.sh

```diff
@@ -86,6 +86,16 @@ apply_template() {
   ${KUBECTL} apply --filename "${CWD}/../generated/${template}"
 }
 
+generate_templates() {
+  [[ ${SKIP_INGRESS} == true ]] && make_template templates/ingress
```

Comment:
`||`?

Reply:
Good catch. I just realize why my code worked. Commit d347d737e561e4328ea5b5c321aae9f46be4f125 makes this a config value. However, I failed to update skip-ingress logic here. I'll get it all fixed and do a new push

Reply:
I just did a "git push -f" to clean fix the original commit in this PR and also fix the SKIP_INGRESS logic